server/controllers/finance/reports/balance_sheet/index.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * Balance sheet Controller
+ *
+ * This controller is responsible for processing
+ * the balance sheet (bilan) report.
+ *
+ * @module reports/balance_sheet
+ *
+ * @requires lodash
+ * @requires lib/db
+ * @requires lib/ReportManager
+ * @requires lib/errors/BadRequest
+ */
+
+const _ = require('lodash');
+const moment = require('moment');
+const db = require('../../../../lib/db');
+const ReportManager = require('../../../../lib/ReportManager');
+
+// report template
+const TEMPLATE = './server/controllers/finance/reports/balance_sheet/report.handlebars';
+
+const ASSET = 1;
```

Comment:
In the future these should be defined as system wide variables - can you create an issue to track this feature?